docs/examples/Smooth_Platform_Coordinate_Transitions.py

```diff
@@ -203,3 +210,136 @@
         color='blue',
         linewidth=3)
 fig
+
+
+# %%
+# Constant Jerk
+# ^^^^^^^^^^^^^
+# The next method is :meth:`~simulator.ConstantJerkSimulator.create_models`
+#
+# This method takes a series of states with cartesian kinematic state space elements and returns a
+# chain (list) of :class:`~.ConstantJerkSimulator` transition models and transition times.
+#
+# The :class:`~.ConstantJerkSimulator` type of transition model is an implementation of a
+# noiseless, constant jerk transition in the kinematic, cartesian subspace of the state space.
+#
+# The user will need to provide both initial and final position and velocities. This can be of any
+# dimension. For example, constant jerk in the space :math:`(x, vx, y, vy)` and
+# :math:`(x, vx, y, vy, z, vz`)` are valid (elements need not be in this order, as a position
```

Comment:
I suggest `(x, \dot{x}, y, \dot{y}, z, \dot{z})`. Also there is an extra ` before the close bracket.